api/src/main/java/org/openmrs/module/registrationcore/api/impl/PatientActionListener.java

```diff
@@ -46,6 +64,19 @@ public void run() {
             }
         }, daemonToken);
     }
+    /**
```

Comment:
Do you wanna put the method description?

Reply:
added